extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -108,96 +114,114 @@ public function execute() {
 	 */
 	protected function reinitialize() {
 		$this->nolang_txt = [];
+
 		return $this;
 	}
 
 	/**
 	 * Provides an array of outbound links from the current document to other doc IDs.
 	 * Filters out self-links (e.g. Edit and the like)
+	 *
 	 * @return array
 	 */
 	protected function getOutboundLinks() {
 		$service = $this->getService();
 		$result = [];
 		$docId = $this->getCurrentDocumentId();
 		if ( $service->getGlobal( 'BacklinksEnabled' ) ) {
-			$backlinks = (new \Wikia\Search\Hooks)->popLinks();
+			$backlinks = ( new \Wikia\Search\Hooks )->popLinks();
 			$backlinksProcessed = [];
 			foreach ( $backlinks as $backlink ) {
-				if ( substr_count( $backlink, $docId.' |' ) == 0 ) {
+				if ( substr_count( $backlink, $docId . ' |' ) == 0 ) {
 					$backlinksProcessed[] = $backlink;
 				}
 			}
 			$result = [ 'outbound_links_txt' => $backlinksProcessed ];
 		}
+
 		return $result;
 	}
 
 	/**
 	 * Optionally sets language field for field. Old backend already does this.
+	 *
 	 * @param string $field
+	 *
 	 * @return string
 	 */
 	protected function field( $field ) {
-		return $this->getService()->getGlobal( 'AppStripsHtml' ) ? (new Utilities)->field( $field ) : $field;
+		return $this->getService()->getGlobal( 'AppStripsHtml' ) ? ( new Utilities )->field( $field ) : $field;
 	}
 
 	/**
 	 * Wraps logic for creating the initial result array, based on which implementation we're using.
 	 * The old version strips HTML from the backend; the new version strips HTML within the IndexService.
+	 *
 	 * @param array $response
+	 *
 	 * @return array
 	 */
 	protected function getPageContentFromParseResponse( array $response ) {
 		$html = empty( $response['parse']['text']['*'] ) ? '' : $response['parse']['text']['*'];
 		if ( $this->getService()->getGlobal( 'AppStripsHtml' ) ) {
 			return $this->prepValuesFromHtml( $html );
 		}
-		return [ 'html' => html_entity_decode($html, ENT_COMPAT, 'UTF-8') ];
+
+		return [ 'html' => html_entity_decode( $html, ENT_COMPAT, 'UTF-8' ) ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "UTF-8" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)